templates/All/Default.html

```diff
@@ -1,44 +1,36 @@
 <div class="coveo-result-frame">
-  <div class="coveo-result-row">
-    <div class="coveo-result-cell" style="width:85px;text-align:center;padding-top:7px;">
-      <span class="CoveoIcon">
-      </span>
-      <div class="CoveoQuickview">
+  <div class="coveo-result-cell" style="vertical-align:top;text-align:center;width:32px;">
```

Comment:
To be honest I believe at some point we should review these whole classes coveo-row, coveo-cell, etc. and make it a layout framework instead using `display:flex` all the way.

But we can't really until we drop IE11 I think. Flex is only partial support :(

Could be an idea to have a `coveo-flex-result-frame` also on top of what we already have and it could be easier to control result layouts using flex.